src/components/Editor/Editor.css

```diff
@@ -134,7 +134,6 @@ html[dir="rtl"] .editor-mount {
   user-select: none;
 }
 
-
```

Comment:
Deleted this empty line to fix this lint error:
<img width="574" alt="screen shot 2017-02-23 at 9 05 32 am" src="https://cloud.githubusercontent.com/assets/4562118/23240412/58d60136-f9a8-11e6-98bb-73c967685c91.png">